frontend/src/metabase/writeback/utils.ts

```diff
@@ -72,6 +72,10 @@ export function isActionDashCard(
   return isActionCard(virtualCard as SavedCard);
 }
 
+export const isLinkActionDashCard = (dashCard: BaseDashboardOrderedCard) =>
+  isActionDashCard(dashCard) &&
+  dashCard.visualization_settings?.click_behavior?.type === "link";
+
```

Comment:
I wonder if we should use slightly different naming and avoid the word "action" in this case? It's actually a "button" viz with a custom destination click behavior without any action attached. Maybe something like `isButtonLinkDashCard` would be better?